src/main/java/io/gravitee/tracer/opentelemetry/exporter/BaseGrpcExporter.java

```diff
@@ -0,0 +1,389 @@
+                                    if (loggedUnimplemented.compareAndSet(false, true)) {
+                                        logUnimplemented(internalLogger, type, statusMessage);
+                                    }
+                                } else if (status == GrpcStatus.UNAVAILABLE) {
+                                    logger.log(
+                                        Level.SEVERE,
+                                        "Failed to export " +
+                                        type +
+                                        "s. Server is UNAVAILABLE. " +
+                                        "Make sure your collector is running and reachable from this network. " +
+                                        "Full error message:" +
+                                        statusMessage
+                                    );
+                                } else {
+                                    if (status == null) {
+                                        logger.log(
+                                            Level.WARNING,
+                                            "Failed to export " + type + "s. Server responded with error message: " + statusMessage
+                                        );
+                                    } else {
+                                        logger.log(
+                                            Level.WARNING,
+                                            "Failed to export " +
+                                            type +
+                                            "s. Server responded with " +
+                                            status.code +
+                                            ". Error message: " +
+                                            statusMessage
+                                        );
+                                    }
+                                }
+                            }
+
+                            private void logUnimplemented(Logger logger, String type, String fullErrorMessage) {
+                                String envVar;
+                                switch (type) {
+                                    case "span":
+                                        envVar = "OTEL_TRACES_EXPORTER";
+                                        break;
+                                    case "metric":
+                                        envVar = "OTEL_METRICS_EXPORTER";
+                                        break;
+                                    case "log":
+                                        envVar = "OTEL_LOGS_EXPORTER";
+                                        break;
+                                    default:
+                                        throw new IllegalStateException(
+                                            "Unrecognized type, this is a programming bug in the OpenTelemetry SDK"
+                                        );
+                                }
```

Comment:
```suggestion
                                String envVar =
                                    switch (type) {
                                        case "span" -> "OTEL_TRACES_EXPORTER";
                                        case "metric" -> "OTEL_METRICS_EXPORTER";
                                        case "log" -> "OTEL_LOGS_EXPORTER";
                                        default -> throw new IllegalStateException(
                                            "Unrecognized type, this is a programming bug in the OpenTelemetry SDK"
                                        );
                                    };
```
New syntax adds readability, wdyt ?